vcluster/manage/certificates.mdx

```diff
@@ -0,0 +1,321 @@
+  ca.crt                       | kubernetes                    | kubernetes     | Jul 12, 2035 19:24 UTC | OK
+  client-ca.crt                | kubernetes                    | kubernetes     | Jul 12, 2035 19:24 UTC | OK
+  ca.crt                       | etcd-ca                       | etcd-ca        | Jul 12, 2035 19:24 UTC | OK
+  healthcheck-client.crt       | kube-etcd-healthcheck-client  | etcd-ca        | Jul 14, 2026 19:24 UTC | OK
+  peer.crt                     | vcluster                      | etcd-ca        | Jul 14, 2026 19:24 UTC | OK
+  server.crt                   | vcluster                      | etcd-ca        | Jul 14, 2026 19:24 UTC | OK
+  front-proxy-ca.crt           | front-proxy-ca                | front-proxy-ca | Jul 12, 2035 19:24 UTC | OK
+  front-proxy-client.crt       | front-proxy-client            | front-proxy-ca | Jul 14, 2026 19:24 UTC | OK
+  server-ca.crt                | kubernetes                    | kubernetes     | Jul 12, 2035 19:24 UTC | OK
+
+```
+
+## Rotating client and server leaf certificates
+
+The command below does the following
+- Creates a backup of the old PKI directory.
+- Rotates the client and server leaf certificates.
+- Restarts the given virtual cluster.
+
+The restart is necessary in order for the control-plane components to use the
+newly created certificates. Currently running workload should not be affected
+by this.
+
+<InterpolatedCodeBlock
+  code={`vcluster -n [[VAR:VCLUSTER NAMESPACE:my-vcluster]] certs rotate [[VAR:VCLUSTER NAME:my-vcluster]]`}
+  language="bash"
+/>
+
+The client and server leaf certificate rotation is less intrusive compared to
+the [CA rotation](#rotating-ca-certificates) explained below. This means that
+connecting to the virtual cluster with e.g. previously issued kubeconfigs
+should continue to work.
+
+## Rotating CA certificates
+
+:::warning
+A CA rotation completely replaces the trust anchor of all certificates. So this
+command must be used with caution. Though, if anything goes wrong the previous
+PKI can be restored from the backup. The steps for this are explained in the
+[section below](#restoring-the-previous-ca-from-a-backup).
+:::
+
+The command below does the following:
+- Creates a backup of the old PKI directory.
+- Rotates all PKI related certificates including the CA certificate itself.
+- Restarts the virtual cluster.
+
+<InterpolatedCodeBlock
+  code={`vcluster -n [[VAR:VCLUSTER NAMESPACE:my-vcluster]] certs rotate-ca [[VAR:VCLUSTER NAME:my-vcluster]]`}
+  language="bash"
+/>
+
+As this replaces the trust anchor completely everything that is configured to
+trust the old CA will effectively be invalidated. E.g. previously issued
+kubeconfigs.
+
+## Special cases
+
+:::warning
+All steps shown below should be done with caution. Please double-check the
+commands before executing them.
+:::
+
+This section describes certain special cases that needs additional manual
+intervention.
+
+### Restoring the previous CA from a backup
```

Comment:
Might make sense to make this easier in the future e.g. with something like `vcluster certs restore`?! Though, don't know how prominent this case should be (cc @cbron @deniseschannon).